tests/test_utils.py

```diff
@@ -424,10 +424,10 @@ def test_perform_request_add_motd_error_insufficient_credentials() -> None:
         "email": "",
     }
     with Mocker() as mock:
-        mock.post(DDSEndpoint.ADD_NEW_MOTD, status_code=403, json=response_json)
+        mock.post(DDSEndpoint.MOTD, status_code=403, json=response_json)
```

Comment:
Not related to this, but instead line 335 and 336 I believe (the test that fails)
Change the two asserts to only:
```
assert str(exc_info.value) == "[Errno Expecting value] str: 0"
```